src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/Fucker.kt

```diff
@@ -46,17 +47,18 @@ object Fucker : Module("Fucker", ModuleCategory.WORLD) {
     private val range by FloatValue("Range", 5F, 1F..7F)
 
     private val action by ListValue("Action", arrayOf("Destroy", "Use"), "Destroy")
-    private val surroundings by BoolValue("Surroundings", true)
-        private val instant by BoolValue("Instant", false) { action == "Destroy" || surroundings }
+    private var surroundings by BoolValue("Surroundings", true)
+    private val hypixelMode by BoolValue("Hypixel-Bypass", false)
```

Comment:
Ah yes jut put always shown value above value that depends on above 2 values why not.

Reply:
my bad

Reply:
idea code auto clean

Reply:
By putting 1 line space between hypixel-bypass and instant, the instant is still below, separated by totally independent value?

Reply:
okok, My translation software misinterpreted your meaning, I thought you were talking about sequencing